src/coda/apps/invoices/urls.py

```diff
@@ -59,4 +60,5 @@
     path("conversions/<int:pk>/edit/", edit_conversion_row, name="edit_conversion"),
     path("conversions/<int:pk>/update/", update_conversion, name="update_conversion"),
     path("conversions/<int:pk>/delete/", delete_conversion, name="delete_conversion"),
+    path("<int:pk>/pay_invoice/", pay_invoice, name="pay_invoice"),
```

Comment:
let's call this endpoint pay instead of pay_invoice as it is already reachable under the `/invoices/` namespace

Reply:
only the path. the name can stay the same